### DISCUSSION TITLE
Welcome to Azure-DevOps Discussions!

### DISCUSSION THREAD

**Discussion by vinijmoura** [vinijmoura/Azure-DevOps#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by staycoolkhan** [comment]
Thanks for such amazing repo. I'm having this error message while running the scripts in powershell "New-Object : Exception calling ".ctor" with "1" argument(s): "Keyword not supported: '"server'." + $sqlcc = New-Object -TypeName System.Data.SqlClient.SqlConnection -Ar ...
+          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [New-Object], MethodInvocationException
    + FullyQualifiedErrorId : ConstructorInvokedThrowException,Microsoft.PowerShell.Commands.NewObjectCommand


Also this is the powershell command I'm running .\Processes.ps1 -PAT "pattokexyz" -Organization "orgnisationxyz"  -Connstr "Server=tcp:mydev.database.windows.net,1433;Initial Catalog=|azuredevopsreports|;Persist Security Info=False;User ID=devuser;Password=devpassword;MultipleActiveResultSets=False;Encryt=True;TrustServerCertificate=False;Connection Timeout=30;"

**Reply by vinijmoura** [reply]
Please contact me on email
vinijmoura@hotmail.com

**Reply by staycoolkhan** [reply]
Thanks @vinijmoura . I have sent you email regarding my above issue.

**Comment by rrober07** [comment]
Can your Azure DevOps report project be used with Azure DevOps 2020?

**Reply by vinijmoura** [reply]
Yes! Normally, before _apis, put {server:port}/tfs/{collection}
I haven't run all the scripts for the server yet

**Reply by rrober07** [reply]
great! thank you!!

**Reply by rrober07** [reply]
Sorry another question... after I go through your process of creating SqlServer DB and extracting the data and reports... do you have a process to keep the data up to date? or do I have to run the drop/create reload when I want updated data to report? or am I missing something?  thanks

**Comment by vinijmoura** [comment]
Exactly. That's why I created a DROP script and recreated it every time!
Example: I can clone this repo and put this PowerShell on nightly pipeline :-)

**Reply by rrober07** [reply]
Awesome! I think I got it now...much appreciated!!
